### ISSUE TITLE
Save Position of Bunkers

### ISSUE THREAD

**Issue by newboerg** [A3Antistasi/A3-Antistasi#189]
Mods: RHS

The Bunkers loose their position after loading persistend savegame
![arma 3 03 11 2018 16_08_35](https://user-images.githubusercontent.com/7376421/47953857-33594b80-df83-11e8-9813-cd828dfde1c8.png)

**Comment by Barbolani77** [comment]
yes, it is a known bug, but the reason for this is still unknown

**Comment by geekwithbadge** [comment]
Barbolani77, first off. Great GREAT script. We LOVE your stuff.  

You ever check out inidbi2 for saving variables instead of the namespace?
